UI.tsv

```diff
@@ -358,7 +358,7 @@ UI_20150317_000357	{@st43}{s24}Market{/}
 UI_20150317_000358	{@st42b}Filter{/}
 UI_20150317_000359	{@st42b}Type{/}
 UI_20150317_000360	{@st42b}Req. Lv{/}
-UI_20150317_000361	{@st42b}Enhance Lv{/}
+UI_20150317_000361	{@st42b}Refinement Lv{/}
```

Comment:
Old "Enhance" is better. If we want to change it it's best to do it in-game to avoid UI conflict.